Terraform/Home_Task_2/modules/alb/main.tf

```diff
@@ -3,40 +3,30 @@ resource "aws_lb" "alb" {
   name               = "lb-terraform"
   internal           = false
   load_balancer_type = "application"
-  ip_address_type    = "ipv4"
-  security_groups    = [var.alb_sg]
+  security_groups    = [aws_security_group.alb.id]
   subnets            = [var.public_subnet_1_id, var.public_subnet_2_id]
 
-  enable_deletion_protection = false
-  /*
-  access_logs {
-    bucket  = aws_s3_bucket.lb_logs.bucket
-    prefix  = "test-lb"
-    enabled = true
-  }
-*/
   tags = {
     Environment = "production"
     Name        = "alb"
   }
 }
 
 resource "aws_alb_target_group" "this_tg" {
-  name        = "terraform-alb-target"
-  port        = 80
-  protocol    = "HTTP"
-  vpc_id      = var.vpc_id
-  target_type = "instance" #default
-  stickiness {             # optional
-    type = "lb_cookie"
-  }
+  name     = "terraform-alb-target"
+  port     = 80
+  protocol = "HTTP"
+  vpc_id   = var.vpc_id
+  #target_type = "instance" #default
+
   health_check {
-    path                = "/index.html"
-    port                = 80
-    protocol            = "HTTP"
-    timeout             = 5
+    #path                = "/index.html"
+    #port                = 80
+    #protocol            = "HTTP"
+    #timeout             = 5
     healthy_threshold   = 5

```

Comment:
healthy_threshold, unhealthy_threshold, interval - define values of these parameters in modules/alb/variables.tf file